clang/lib/Tooling/DependencyScanning/DependencyScanningCASFilesystem.cpp

```diff
@@ -72,10 +72,7 @@ storeDepDirectives(cas::ObjectStore &CAS,
     TokenIdx += Directive.Tokens.size();
   }
 
-  auto Blob = CAS.createProxy(std::nullopt, Buffer);
-  if (!Blob)
-    return Blob.takeError();
-  return Blob->getRef();
+  return CAS.storeFromString(std::nullopt, Buffer);
```

Comment:
Thoughts: maybe we should rename this method just to `store` if it is more commonly used than the other `store` method and `storeFromString` name doesn't provide any value in terms of clarity.

Reply:
Yeah, I like this idea. The other one is harder to use, it shouldn't get the shorter name; we should probably rename
* `store` -> `storeFromArray`
* `storeFromString` -> `store`

via a separate PR.

Reply:
I would even argue that have both `store(StringRef)` and `store(ArrayRef<char>)` won't affect the clarity of the code. They can both be called `store`.